packages/kbn-monaco/src/esql/index.ts

```diff
@@ -6,7 +6,8 @@
  * Side Public License, v 1.
  */
 
+import { LangModule as LangModuleType } from '../types';
 import { ID } from './constants';
 import { lexerRules } from './lexer_rules';
 
-export const EsqlLang = { ID, lexerRules };
+export const EsqlLang = { ID, lexerRules } as LangModuleType;
```

Comment:
nit; could we update this to:

```suggestion
export const EsqlLang: LangModuleType = { ID, lexerRules };
```

Reply:
Ok, not a problem. I was using the style of @kbn/monaco )

Reply:
Done.